demo/mmediting_inference_tutorial.ipynb

```diff
@@ -0,0 +1,346 @@
+    "\n",
+    "This is a quick guide for you to infer with existing models. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install MMEditing\n",
+    "Please refer to [README.md](README.md) for installation instruction."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Inference API Introduction\n",
+    "\n",
+    "You could use inference api in your python code or with command line."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.1 Inference with python code\n",
+    "\n",
+    "MMEditing inference api makes it easy to infer your own image or video with two lines python code. \n",
+    "\n",
+    "Take image translation for example.\n",
+    "\n",
+    "There are two steps:\n",
+    "\n",
+    "First, create a MMEdit instance by a pretrained model name.\n",
+    "\n",
+    "Second, infer your own image with this MMEdit instance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mmedit.edit import MMEdit\n",
+    "\n",
+    "# Create a MMEdit instance\n",
+    "editor = MMEdit('pix2pix')\n",
+    "# Infer a image. Input image path and output image path is needed.\n",
+    "editor.infer(img='resources/input/translation/gt_mask_0.png', result_out_dir='resources/demo_results/tutorial_translation_res.jpg')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.2 Inference with command line\n",
+    "\n",
+    "There is a command line interface in this folder (MMEditing/demo/mmediting_inference_demo.py).\n",
+    "\n",
+    "You could infer a model with this interface like this (do this in MMEditing root path)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "bash demo/mmediting_inference_demo.py --model-name pix2pix --img='resources/input/translation/gt_mask_0.png', result_out_dir='resources/demo_results/tutorial_translation_res.jpg'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Infer with models of different tasks\n",
+    "\n",
+    "There are multiple task types in MMEditing: conditional, inpainting, matting, restoration, translation, unconditional, video_interpolation, video_restoration. \n",
+    "\n",
+    "We provide some models for each task. All available models could be printed out like this."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mmedit.edit import MMEdit\n",
+    "\n",
+    "# print all available models for inference.\n",
+    "inference_supported_models = MMEdit.inference_supported_models\n",
```

Comment:
can we provide `inference_supported_task()`

Reply:
btw, you need to correct the value of task in metafile

Reply:
good idea. this function is supported. resolved

Reply:
> btw, you need to correct the value of task in metafile

value of task in metafile fixed